src/base/projects/Header.svelte

```diff
@@ -134,29 +134,29 @@
     on:click={() => toggleContent(ProjectContent.History, true)}>
     <strong>{tree.stats.commits}</strong> commit(s)
   </div>
-  {#if project.issues > 0}
+  {#if project.issues !== undefined}
```

Comment:
Don't really like checking against `undefined` in application logic; what's the idea here?

Reply:
Ok I got it, can you do `if "issues" in project`? would that work?

Reply:
hmm yeah that should work

Reply:
Ok I think we should be able to resolve it also with `if (project.issues)` since we only have to rule out that it's not `undefined`.

Reply:
Done in [e7dcd40](https://github.com/radicle-dev/radicle-interface/pull/333/commits/e7dcd408fe68d87b3c9e5020e2ef165f04c2a5b4)